armory/docker/management.py

```diff
@@ -80,7 +80,7 @@ def exec_cmd(self, cmd: str, user="", expect_sentinel=True) -> int:
         #  ie after the sentinel
         sentinel_found = False
         for out in result.output:
-            output = out.decode().strip()
+            output = out.decode(encoding="utf-8", errors="replace").strip()
```

Comment:
I'm a little reluctant to use `errors="replace"` if just guarding against potential, not actual, errors. "Errors should never pass silently."

Reply:
This handles 50% of use cases, the other 50 are ä̷̢͙̐̆͐̈́͋͌̓̄́̆͛͘̕r̶̼̮̠̳͓̣̳̯̯̭͚͂̾̓̈́ẽ̴̤̻͖̣̫̞̾̃ ̴̨͖̗̤̱̥̻̣͕̣͇̿̽͋͗̐͜ą̷͓̜̻̥͓̠̺̼̏͝n̴͍̪̜̩̮̟͙̪̤̦͈̼̮̾̏̌̿̑̑̎̒ ̵͚͎̯̥̮̣͈̗͙̯̼͉̰̍̐̎̎e̵̻̞̦̘̾͌̍ỷ̶͙̫̤͍͇͖̣̓͂͜͝ͅḛ̴̭̎̿̄̈̓̃̚͝͝ľ̷͍̭̤̤̼̞͉̥̰͌͜͜ė̵̪́̈́̉̓́͗̈́͊̾͌̈́͜͝͝s̵̨̛͉̩̭͖̣̟͙͕̙̞͖͔͛͑̊̀́̀̐̒̀̈́̉͗̏̈ş̵̻͍͔̑̀̈́̉͌̓̽͌͐̒̚ ̴̛͍̿̀̓́͌̾̈́̔͋̊̃a̶̭͚̪̒͒̓̈̑́b̸͔̈́͌o̴̢̡̰͇͈̠̖̳͉͚̩̭̥̽̓̈́͊̈́̈́͛̓̓͂m̷̡͈͚̤̓͋̓̾̿͘͠ȉ̸̮͇̙̏̂̍̔͑͒̕̚͠n̶̡̥͉̺̝̼̖̭̺̳͕̳̅̏̅͐̌͐̔͂̾̓͌̄͒̚͜͠ą̷̨̝̖̺̥͂̓̽̓͛̑̾̈̕͘̕͠͝t̶̡̡͖̹̥̥̿̾̏̄́́̎̆̾͆̏͝͠ȉ̸̬͉̹̜̰͕̣̑̄͛̆͒̇̔̽͐̚̕͝͝ȍ̸̢̡̭̯͔̙̘͇͙̝̜͚̀̀́̃͒̾̕͠ņ̵̣̟̥̖̩̗̣̻̀̀̇͋̕.